src/icon/rotateIcon.tsx

```diff
@@ -0,0 +1,20 @@
+// tslint:disable:max-line-length
+import React from 'react'
+import BaseIcon from '_utils/icon'
+import { BaseIconDefaultProps } from '_utils/icon/BaseIcon'
+
+export const RotateIcon = (props: Icon) => (
+  <BaseIcon {...props}>
+    <g fill="none" fillRule="evenodd" strokeLinecap="round" strokeLinejoin="round">
```

Comment:
I get an even lighter result with svgomg (most notably, you can use one single `stroke`)
`<g fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round" stroke="#708C91">
    <path d="M19.5 10c0 5.2-4.3 9.5-9.5 9.5A9.6 9.6 0 01.5 10C.5 4.8 4.8.5 10 .5c3.7 0 7 2 8.5 5.2"/>
    <path d="M19.3.2l-.7 5.6-5.7-.8"/>
</g>`